recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -20,60 +18,52 @@ class MsdfAtlasGenConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     package_type = "application"
 
-    generators = "CMakeToolchain"
-    exports_sources = ["CMakeLists.txt"]
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def requirements(self):
         self.requires("artery-font-format/1.0")
         self.requires("msdfgen/1.9.1")
+        self.requires("lodepng/cci.20200615")
```

Comment:
I had to add this require because the original recipe had a bug and was adding the include for this lib from the vendored version in the repo of msdf-atlas-gen instead of the conan 1.